src/SprykerEco/Yves/FactFinderWebComponents/Form/WebComponentConfigToTwigConfigTransformer.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\FactFinderWebComponents\Form;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+class WebComponentConfigToTwigConfigTransformer implements DataTransformerInterface
+{
+    public const ITEMS_PARAMETER = 'items';
+
+    /**
+     * @inheritdoc
+     *
+     * @param mixed $value
+     *
+     * @return mixed|void
+     */
+    public function transform($value)
+    {
+        $result = [];
+
+        if (is_array($value) == false) {
+            return $result;
+        }
+
+        $result = $value;
+        if (isset($value['properties'])) {
+            $result['component'] = $this->transformFromArrayToHtmlAttributes($value['properties']);
+            unset($result['properties']);
+        }
+
+        return $result;
+    }
+
+    /**
+     * @param array $componentParameters
+     *
+     * @return string
+     */
+    protected function transformFromArrayToHtmlAttributes(array $componentParameters): string
+    {
+        $htmlAttributesString = '';
+
+        foreach ($componentParameters as $itemKey => $itemValue) {
+            if (is_bool($itemValue) == false) {
```

Comment:
I think you can may not use comparison here, just `!is_bool($itemValue)` or at least use strict comparison.

Reply:
Replaced with a strict comparison.